api/campaigns/index.js

```diff
@@ -89,6 +90,23 @@ router.get('/camp/:id', (req, res) => {
     });
 });
 
+router.get('/:id/submissions', async (req, res) => {
+  
+  const { id } = req.params;
+
+  ApplicationSubmissions.findByCampaignId(id)
```

Comment:
Why no `await`?

If you don't use `await` you at least need to return this Promise.